catalogs/usfimr-s1/s3_stac_io.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
```

Comment:
Huh. Surprised this doesn't exist in pystac already. I guess because of the boto3 dependency.